setup.cfg

```diff
@@ -1,3 +1,41 @@
+[metadata]
+name = pysat
+version = file: pysat/version.txt
+url = https://github.com/pysat/pysat
+author = Russell Stoneback
+author_email = rstoneba@utdallas.edu
+description = 'Supports science data analysis across measurement platforms'
+keywords =
+  pysat
+  ionosphere
+classifiers =
+  Development Status :: 5 - Production/Stable
+  Intended Audience :: Science/Research
+  Topic :: Scientific/Engineering :: Astronomy
+  Topic :: Scientific/Engineering :: Physics
+  Topic :: Scientific/Engineering :: Atmospheric Science
+  License :: OSI Approved :: BSD License
+  Natural Language :: English
+  Programming Language :: Python :: 3.6
+  Programming Language :: Python :: 3.7
+  Programming Language :: Python :: 3.8
+  Operating System :: MacOS :: MacOS X
+  Operating System :: POSIX :: Linux
+license_file = LICENSE
+long_description = file: description.txt
```

Comment:
add an issue to ultimately use a scrubbed version of readme.md here

Reply:
Added this to #607